ivy/functional/ivy/extensions.py

```diff
@@ -453,3 +453,43 @@ def sinc(
 
     """
     return ivy.current_backend(x).sinc(x, out=out)
+
+
+@integer_arrays_to_float
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+def collapse_repeated(
```

Comment:
Only tensorflow has this function, it has to be implemented compositionally using other Ivy functions.